rust/src/datatypes.rs

```diff
@@ -15,14 +15,32 @@
 // specific language governing permissions and limitations
 // under the License.
 
+/*!
```

Comment:
Shouldn't this use `//!` style comments instead?

Reply:
I believe that surrounding with `/*!` and `*/` is equivalent to starting each line with `//!`.  I'm finding it hard to confirm this but I noticed this approach [here](https://github.com/BurntSushi/rust-csv/blob/master/src/lib.rs)

I guess we should stick with `//!` as it seems to be most used approach.